Libraries/Map/Map.cpp

```diff
@@ -174,4 +174,15 @@ void Map::moveSouth() {
 void Map::moveWest() {
   orientation_ = 4;
   --current_column_;
+}
+
+bool Map::tileCandidateToVisit(const uint8_t row, const uint8_t column) {
+  if (map_[row][column].isAccessible()
+  && !map_[row][column].isBlack()
+  && !map_[row][column].isVisited()) {
+    return true;
+  }
+  else {
+    return false;
```

Comment:
very nice! And this could be written with only one return.

Hint: what condition must be true for you to return true?

Reply:
Oooooh I get the point.